ansible/roles/operator-pipeline/templates/openshift/tasks/trigger-marketplace-replication.yml

```diff
@@ -0,0 +1,42 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: trigger-marketplace-replication
+spec:
+  params:
+    - name: bundle_path
+      description: path indicating the location of the certified bundle within the repository
+    - name: git_repo_url
+      description: ssh address of GitHub repository
+    - name: package
+      description: Operator package name
+    - name: pipeline_image
+    - name: ocp_version
+      description: OCP versions in the bundle annotations
+    - name: version
+      description: Version of the operator bundle
+  workspaces:
+    - name: ibm-webhook-token
+    - name: source
+  steps:
+    - name: trigger-marketplace-replication
+      image: "$(params.pipeline_image)"
+      workingDir: $(workspaces.source.path)
+      env:
+        - name: WEBHOOK_TOKEN_PATH
+          value: $(workspaces.ibm-webhook-token.path)/token
+      script: |
+        #! /usr/bin/env bash
+        # Don't use set -x to avoid exposing webhook token
+        set -e
+
+        export IBM_WEBHOOK_TOKEN=$(cat $WEBHOOK_TOKEN_PATH)
```

Comment:
Is it possible to use `secretKeyRef` instead of doing this `export`? Take a look at [this](https://github.com/redhat-openshift-ecosystem/operator-pipelines/blob/main/ansible/roles/operator-pipeline/templates/openshift/tasks/set-github-status.yml#L35-L39) for an example. There is a very real risk of leaking credentials this way so we should try to avoid it if possible.